src/installation/liveimages/guide-chroot.md

```diff
@@ -0,0 +1,388 @@
+# Installation Guide (chroot)
+
+## Pre-installation
+
+### Set the keyboard layout
+
+The default console keymap is US. Available layouts can be listed with:
+
+```
+$ ls /usr/share/kbd/keymaps/*/*/*.map.gz | less
+```
+
+To modify the layout, append a corresponding file name to
+[`loadkeys(1)`](https://man.voidlinux.org/loadkeys), omitting path and file
+extension. For example, to set the Dvorak keyboard layout:
+
+```
+# loadkeys dvorak
+```
+
+### Verify the boot mode
+
+Many new motherboards will support booting in UEFI mode, but some will not,
+instead booting in legacy BIOS mode. You will need to know in which mode
+your system will boot when it is time to
+[partition the disks](#partition-disks).
+
+If you have boot via UEFI mode, the directory `/sys/firmware/efi` should
+exist and be populated.
+
+```
+$ ls /sys/firmware/efi
+```
+
+If you know your system supports booting into UEFI mode, but does not appear to
+have done so, check your UEFI/BIOS settings (can usually be accessed by pressing
+a key like `F12` early in the boot process. Look for settings like *UEFI-mode*
+or *Disable Legacy BIOS mode*.
+
+### Connect to the Internet
+
+The following steps are a high-level overview for connecting to a wireless
+network. For more information, you should consult the
+[Network](./config/network/index.md) page.
+
+Check the available network interfaces and take note of the interface name. In
+this case, it is `wlan0`.
+
+```
+$ ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+	link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: wlan0: <BROADCAST,MULTICAST> mtu 1500 qdisc mq state DOWN mode DEFAULT group default qlen 1000
+	link/ether 1c:4d:70:e0:5d:a3 brd ff:ff:ff:ff:ff:ff
+```
+
+Enable the network interface:
+
+```
+# ip link set <interface> up
+```
+
+Start the DHCP server:
+
+```
+# dhcpcd <interface>
+```
+
+Use [`wpa_passphrase(8)`](https://man.voidlinux.org/wpa_passphrase.8) to create
+a network configuration, and append it's output to the file
+`/etc/wpa_supplicant.conf`. See
+[`wpa_supplicant.conf(5)`](https://man.voidlinux.org/wpa_supplicant.conf.5) for
+more options.
+
+```
+# wpa_passphrase "SSID" "passphrase" >> /etc/wpa_supplicant.conf
+```
+
+Start [`wpa_supplicant(8)](https://man.voidlinux.org/wpa_supplicant.8):
+
+```
+# wpa_supplicant -B -i <interface>
+```
+
+Verify that you are connected to the internet:
+
+```
+ping voidlinux.org
+```
+
+Note that you can terminate a running command by sending a `SIGINT` signal with
+`ctrl-c`.
+
+### <a name="partition-disks"></a>Partition the disks
```

Comment:
Partitioning is really variable. I'm tempted to just have a separate partitioning section common to all install guides.

Reply:
I was thinking the same thing.

Reply:
Agreed. I think it's really important for users to understand the options they have and how to make the right choices. Which section should partitioning be moved to?

Reply:
For now you could keep it here and somebody could move it on a second pass, or you could make a partitioning subsection in the install section.

Reply:
Well if you would like to add a section in your PR, that is fine too. But I think it may be easier to leave the partitioning sections as is and combine them with a second pass.

Reply:
Let's go with that for now.

Reply:
Will resolve for now, until we comb it through again.